src/content/details/faqs.mdx

```diff
@@ -166,9 +165,9 @@ hyperparameters are fully under your control and allow you the flexibility to ea
 
 **Do you support INT8 and INT16 (quantized) operations?**
 
-The DeepSparse Engine runs at FP32 and has support for INT8.  With Intel Cascade Lake generation chips and later,
+DeepSparse runs at FP32 and has support for INT8.  With Intel Cascade Lake generation chips and later,
 Intel CPUs include VNNI instructions and support both INT8 and INT16 operations. On these machines, performance improvements
-from quantization will be greater. The  DeepSparse Engine has INT8 support for the ONNX operators QLinearConv, QuantizeLinear,
+from quantization will be greater. The  DeepSparse has INT8 support for the ONNX operators QLinearConv, QuantizeLinear,
```

Comment:
Are you saying that all four components (Benchmark, Engine, Pipelines, Server) has INT8 support for all these operators listed or is it really the DeepSparse Engine component has support....